### PR TITLE
Add port range info

### LINKED ISSUE #7: Add PortRangeInfo

### PR DESCRIPTION
**Pull request by chrisrohr**
Closes #7

### CODE REVIEW

File: src/main/java/org/kiwiproject/dropwizard/util/startup/PortRangeInfo.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package org.kiwiproject.dropwizard.util.startup;
+
+import static com.google.common.base.Preconditions.checkState;
+import static org.kiwiproject.base.KiwiPreconditions.requireValidPort;
+
+import lombok.Getter;
+
+@Getter
+public class PortRangeInfo {
```
Comment (sleberknight) [inline src/main/java/org/kiwiproject/dropwizard/util/startup/PortRangeInfo.java:9]: Do we want to maybe change this to `AllowablePortRange`?

Comment (sleberknight) [reply src/main/java/org/kiwiproject/dropwizard/util/startup/PortRangeInfo.java:9]: ...and docs?

Comment (chrisrohr) [reply src/main/java/org/kiwiproject/dropwizard/util/startup/PortRangeInfo.java:9]: sure and sure

Comment (chrisrohr) [reply src/main/java/org/kiwiproject/dropwizard/util/startup/PortRangeInfo.java]: done

File: src/main/java/org/kiwiproject/dropwizard/util/startup/AllowablePortRange.java [hunk 2]
```diff
@@ -5,8 +5,11 @@
 
 import lombok.Getter;
 
+/**
+ * Model to track the allowable port range for a service to bind to
```
Comment (sleberknight) [inline src/main/java/org/kiwiproject/dropwizard/util/startup/AllowablePortRange.java:9]:
```suggestion
 * Defines the allowable port range for a service to bind to, in situations where there is a restriction.
```

**Review by sleberknight — APPROVED** [review]

**Pull request merged**